src/gpodder/util.py

```diff
@@ -1386,23 +1387,33 @@ def bluetooth_send_file(filename):
         return False
 
 
-def format_time(value):
+def format_time(seconds):
     """Format a seconds value to a string
 
     >>> format_time(0)
-    '00:00'
+    '0:00'
     >>> format_time(20)
-    '00:20'
+    '0:20'
     >>> format_time(3600)
-    '01:00:00'
+    '1:00:00'
```

Comment:
Time formatted like this looks horrible. Please leave the output to the [ISO standard](https://en.wikipedia.org/wiki/ISO_8601#Times) time format, as it was.

Reply:
The format_time() function is only used for duration, and not actual timestamps. Any episode with a duration longer than a day will wrap and you might download a 25 hour livestream because it only shows as 01:00:00 or 1:00:00 in length, instead of 25:00:00. I can add the zero padding if you still want it.

Reply:
Ok, showing durations with hours > 23 should be fine, since there are probably not that many cases where this number is above 100. I think the `%02d` format is essential for all H, M, S components though.

Reply:
Fixed.